docs/get-started/celeba.md

```diff
@@ -1,114 +0,0 @@
-# Finetuning Pretrained ResNet for Celebrity Face Search
```

Comment:
I beleive we then need to remove this from TOC as well. And should't we replace these tutorials with something, not just remove them? For celebrity we could fine tune with tuner, and use docarray with image sprites/embedding visualization, as well as maybe evaluator

Reply:
i think we have too many image related tutorials now, i think 1 image, 1 text and 1 pointcloud should be good